kf_lib_data_ingest/etl/ingest_pipeline.py

```diff
@@ -114,6 +118,12 @@ def __init__(
         kwargs = {arg: values[arg] for arg in args[2:]}
         # Don't log anything that might have secrets!
         kwargs.pop('auth_configs', None)
+
+        # Log ingest package dir, ingest lib version, and commit hash
+        kwargs['ingest_package_dir'] = self.ingest_config_dir
+        kwargs['version'] = VERSION
+        kwargs['commit'] = COMMIT_SHA
```

Comment:
Why are these thrown into kwargs instead of using a new call to logger?

Reply:
Idk, I just did it. Could be its own logged message. Is that preferred?